File: src/prog_langs/swift/for.swift
//  : [String]は入れなくても宣言できる
let strings : [String] = ["hoge", "piyo", "huga", "test"]

// 0から1ずつ増やして4回まわす
for i in 0 ..< 4{
    print(strings[i])

    if ( i == 3 ) {
	    print("========配列ここまで========")
	} 
}

// 配列が終わるまで（indexを取得しながら）回す
for (index, string) in strings.enumerated() {
    print(string)
    // 線の表示
    if ( index == 3 ) {
	    print("========配列ここまで========")
	}
}

// 配列が終わるまで回す
for string in strings {
    print(string)
}

File: src/prog_langs/php/for.php
<?php
	$x = ['test','hoge','piyo','huga'];

	for($val = 0; $val <= 3; $val++){// 0から始まり、3になるまで1づつ増やしながら回す、つまり計4回
		echo $x[$val]."\n";
	}
	// 実行結果
	// test
	// hoge
	// piyo
	// huga
?>